src/views/console/Console.ts

```diff
@@ -0,0 +1,24 @@
+import { createInterface } from "readline";
+
+export class Console {
```

Comment:
Im not sure if this class Console, its ok, in this views folder.

Maybe its a helper? Or maybe its, better into infraestructure?

Or maybe... we can rename as ConsoleView, and the other classes, extends this ConsoleView (With this method we dont need to inject or referer it.